easybuild/easyconfigs/d/dcm2niix/dcm2niix-1.0.20180622-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'CMakeMake'
+
+name = 'dcm2niix'
+version = '1.0.20180622'
+
+homepage = 'https://github.com/rordenlab/dcm2niix'
+description = """dcm2niix is a designed to convert neuroimaging data from the DICOM format to the NIfTI format."""
```

Comment:
It looks like there is an `a` too much in there (it is actually a typo in the repo)